### ISSUE TITLE
Console errors because of new content security policy

### ISSUE THREAD

**Issue by ledestin** [thepracticaldev/dev.to#8792]
**Is your feature request related to a problem? Please describe.**

I'm getting a stream of errors in JS console:

pusher.js?782e:6050 Refused to connect to 'wss://ws-us2.pusher.com/app/Optional?protocol=7&client=js&version=5.1.1&flash=false' because it violates the following Content Security Policy directive: "connect-src 'self' https: http://renner:3035 ws://renner:3035".

test-2b3e#/ledestin/test-2b3e/comments/new/15:1 Access to XMLHttpRequest at 'https://sockjs-us2.pusher.com/pusher/app/Optional/113/jb01ugep/xhr_streaming?protocol=7&client=js&version=5.1.1&t=1592606598161&n=7' from origin 'http://renner:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

I've confirmed that if I disable the content security policy, there are no such errors.
https://github.com/thepracticaldev/dev.to/blob/be07697d6d608be21bcd98e621ec09cceae149c3/config/initializers/content_security_policy.rb#L19

**Describe the solution you'd like**

Perhaps, allow ws-us2.pusher.com in the CSP, and see whether the rest of the errors disappear.

**Describe alternatives you've considered**

None.

**Comment by citizen428** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by citizen428** [comment]
Just to clarify, this is for local development, not on the production site, right (based on `rennner:3000`)?

**Comment by ledestin** [comment]
@citizen428 yes, local dev env. The policy was added only for dev env.

**Comment by burakakca** [comment]
I'm getting same error,  is an any improvement there

**Comment by mikeydorje** [comment]
Closing as stale. Local development CSP errors with Pusher websockets from 2020. Last activity August 2021. CSP configuration likely evolved since then.

If this still occurs in current local development, please open a new issue with current error details.
